lib/hexpm_web/templates/layout/header.html.eex

```diff
@@ -12,6 +12,7 @@
 
     <link rel="search" type="application/opensearchdescription+xml" title="Hex" href="/hexsearch.xml">
     <link rel="stylesheet" href="<%= Routes.static_path(Endpoint, "/css/app.css") %>">
+    <link rel="alternate" type="application/rss+xml" title="RSS - Blog" href="/feeds/blog.xml">
```

Comment:
👍

Reply:
Should we do this on every page or only on the blog itself?

Reply:
> Should we do this on every page or only on the blog itself?

I don't have a strong opinion here. I copied the placement of this `link` element from https://github.com/hexpm/hexpm/pull/29/files#diff-3b562e58bfe1a06ce7c378974605cbc8 and have yet to find out how one would include it only on the blog.